server.js

```diff
@@ -1,27 +1,96 @@
-import express from "express";
-import cors from "cors";
-import mongoose from "mongoose";
+/* eslint-disable */
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
-mongoose.Promise = Promise;
+import express from 'express'
+import cors from 'cors'
+import mongoose from 'mongoose'
 
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+mongoose.Promise = Promise
 
-// Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+// Defines the port the app will run on. Defaults to 8080, but can be 
+// overridden when starting the server. For example:
+//
+//   PORT=9000 npm start
+const port = process.env.PORT || 8080
+const app = express()
 
-// Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+const ThoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+    trim: true
+  },
+  hearts: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: () => Date.now()
+  }
+})
+
+const Thought = mongoose.model('Thought', ThoughtSchema)
+
+// ADD MIDDLEWARE
+app.use(cors())
+app.use(express.json())
+
+// ROUTES
+app.get('/', (req, res) => {
+  res.send('Tjaba, this is my API for my frontend app, see it here "https://tiopunkter.netlify.app"')
+})
+
+// ENDPOIJNT SHOWS 20 LATEST MSG's
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  res.status(200).json(thoughts)
+})
```

Comment:
I tried this endpoint in backend  Seems to work fine, All the messages/thoughts is there.